src/components/newProposal/steps/Actions.tsx

```diff
@@ -170,6 +190,13 @@ export const AddActionButton = ({
             <HiCircleStack className="h-5 w-5 shrink-0" />
             <span>Mint tokens</span>
           </DropdownMenuItem>
+          <DropdownMenuItem
+            onClick={() => append(emptyMergeData)}
+            className="gap-x-2 hover:cursor-pointer"
+          >
+            <AiFillGithub className="h-5 w-5 shrink-0" />
```

Comment:
Please use `FaGithub` instead, to be consistent with the usage of icon libraries